.env.backup

```
@@ -0,0 +1,8 @@
+# Auto-generated by Junie for smoke testing
+DEBUG=True
+SECRET_KEY=development-secret-key-for-tests
+ALLOW_MISSING_DATABASES=True
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ALLOW_MISSING_DATABASES key should go before the DEBUG key

(UnorderedKey)